src/style.css

```diff
@@ -71,3 +71,8 @@
   background-color: #95ffcd;
   text-align:center;
 }
+
+footer {
```

Comment:
I see you are redefining the same footer styles here.

How about this? Let's create a file `global.css` that can be used to define styles that are global to all pages. Basically, if you need to style a footer, just link the global stylesheet and use the class. That way, you only define the class .site-footer in one place.